test/integer-field-events.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>vaadin-integer-field events tests</title>
+
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../vaadin-integer-field.html">
+  <link rel="import" href="helpers.html">
+  <link rel="import" href="../../test-fixture/test-fixture.html">
+</head>
+
+<body>
+  <test-fixture id="integer-field">
+    <template>
+      <vaadin-integer-field></vaadin-integer-field>
+    </template>
+  </test-fixture>
+<script>
+describe('events', () => {
+  let integerField, input;
+
+  describe('has-input-value-changed event', () => {
+    beforeEach(async() => {
+      integerField = fixture('integer-field');
+      input = integerField.inputElement;
+    });
+
+    it('should fire the event on input value presence change', async() => {
+      const hasInputValueChangedSpy = sinon.spy();
+      integerField.addEventListener('has-input-value-changed', hasInputValueChangedSpy);
+      input.value = '5';
+      input.dispatchEvent(new CustomEvent('input'));
+      await nextFrame();
```

Comment:
It seems that we can remove these `nextFrame()` from the added tests. They were needed for Lit integration in v24.